File: python/challenges/fizz_buzz.py
def fizz_buzz(n: int) -> str:
    if n % 3 == 0 and n % 5 == 0:
        return "fizzbuzz"
    elif n % 3 == 0:
        return "fizz"
    elif n % 5 == 0:
        return "buzz"
    else:
        return str(n)


def for_range(r: range) -> dict[int, str]:
    result = {}
    for n in r:
        result[n] = fizz_buzz(n)
    return result

File: go/challenges/fizz_buzz/fizz_buzz.go
package fizzbuzz

import "strconv"

func FizzBuzz(n int) string {
	if n%3 == 0 && n%5 == 0 {
		return "fizzbuzz"
	} else if n%3 == 0 {
		return "fizz"
	} else if n%5 == 0 {
		return "buzz"
	} else {
		return strconv.Itoa(n)
	}
}

func Range(from, to int) map[int]string {
	result := make(map[int]string)
	for i := from; i <= to; i++ {
		result[i] = FizzBuzz(i)
	}
	return result
}
